tests/test_file_utils.py

```
@@ -0,0 +1,1078 @@
+                f.flush()
+                files.append(f.name)
+                f.close()
+
+            result = read_files(files)
+            for i in range(3):
+                assert f"content {i}" in result
+        finally:
+            for file_path in files:
+                try:
+                    os.unlink(file_path)
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)